Core1Platform/AdministratingCore1/03_ManageGroups.md

```diff
@@ -0,0 +1,186 @@
+ 
+
+
+## Add user to user group
+
+Add any number of users to a user group to assign them rights subsequently.
+
+#### Prerequisites
+
+
+- A user group has been created, see [Create a user group](#create-a-user-group).
+
+#### Procedure
+
+*Settings > Users and groups > Tab GRUPPENVERWALTUNG (GROUP MANAGEMENT)*
+
+![Group management](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupCreated.png "[Group management]")
+
+1. Select the checkbox of the user group for which you want to add or remove users.   
+    The editing toolbar is displayed.
+
+2. Click the ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit) button in the editing toolbar.       
+    The *Edit group "group name"* view is displayed. By default, the *Users* tab is preselected.
+
+    > [Info] If no users have been added yet, the *No users added. Use + button to add one to group* notice is displayed. Otherwise, the details of the user(s) included in the group are displayed.   
+    
+     ![Edit group no users](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupEditNoUsers.png "[Edit group no users]")
+
+
+3. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.    
+    The *Add user* window is displayed.
+
+    ![Add user](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupAddUser.png "[Add user]")
+
+4. Click the *Add user* drop-down list and select the desired user.   
+    The selected user has been added to the group. Per default, the validity period is set from today to the system end date. 
+
+    ![Add user](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupAddUserValidity.png "[Add user]")
+
+     1. Click the ![Calendar](../../Assets/Icons/Calendar.png "[Add]") (Calendar) button and select the validity start date in the *Valid from* field.  
+
+     2. Click the ![Calendar](../../Assets/Icons/Calendar.png "[Calendar]") (Calendar) button and select the validity end date in the *End date* field.    
+        > [INFO] It is not possible to change the validity period later. If you want to delimit the validity period, you must remove the group assignment and assign the user again with a different validity period.
+
+5. To add further users, click the ![Add](../../Assets/Icons/Plus05.png "[Add]") (Add) button below the *Add user* drop-down list.   
+    A further drop-down list is displayed. Repeat steps **4** and **5** to add as many users as necessary.
+
+6. Click the [SAVE] button in the upper right corner of the *Add user* window.  
+    The users have been saved to the group and are displayed in the list of users.
+
+7. Click the [SAVE] button in the upper right corner of the *Edit group* window.   
+    The *Group saved* pop-up window is displayed. The changes to the group have been saved.
+
+    ![Group saved](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupSaved.png "[Group saved]")
+
+
+## Remove user from a group via the group management
```

Comment:
```suggestion
## Remove a user from a group
```
 "via the group management" würde ich weglassen. Es geht nicht anders, oder?

Reply:
ne es geht schon anders, sonst hätte ich es nicht so geschrieben